src/server/pages/index.js

```diff
@@ -10,7 +10,23 @@ export default function renderPage (req, res) {
 
   res.setHeader('Content-Type', 'text/html')
   function send (html) {
-    res.send(`<!DOCTYPE html><head><style type="text/css">${css()}</style><meta name="viewport" content="width=device-width, initial-scale=1"></head><body class="__next-auth-theme-${theme}"><div class="page">${html}</div></body></html>`)
+    res.send(`<!DOCTYPE html>
```

Comment:
So obviously we could generate this smarter than an inline string... But for now - since it is such simple code - I think we can live with it. The reason it was on a simple line is (I guess) to make the sent html as small as possible. Could you remove the white spaces to reflect the code that was here before?

Reply:
Sure. I just thought it would be more readable if its multi-line.  
But as you wish.

Reply:
It is certainly, and we should maybe create a new .html file so we get syntax highlighting and so on, but this could be a different PR. Than we should also minify that file and inject variables into it. It's just more work not indeed for this PR, but ideas are welcome! :)